ui/lib/utils/TiDBSQLFormatter/formatSql.ts

```diff
@@ -0,0 +1,7 @@
+import TiDBSQLFormatter from './TiDBSQLFormatter'
```

Comment:
Maybe the directory and file can be `utils/sqlFormatter/index.ts`

Then the usage can be as simple as: import formatSql from '@lib/utils/sqlFormatter`.

Reply:
Yep, that's much better.